packages/config/config/devices/0x0086/templates/aeotec_template.json

```diff
@@ -3298,5 +3306,21 @@
 				"value": 2
 			}
 		]
+	},
+	"lock_configuration": {
+		"label": "Manual button press and parameter setting lock",
+		"valueSize": 1,
+		"defaultValue": 0,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disabled",
+				"value": 0
+			},
+			{
+				"label": "Enabled",
+				"value": 1
+			}
+		]
 	}
```

Comment:
This template already exists

Reply:
switched to master_template.json#base_enable_disable now, thanks